webapp/Connector/src/view/FilterStatus.js

```diff
@@ -15,11 +15,80 @@ Ext.define('Connector.view.FilterStatus', {
 
     cls: 'filterstatus',
 
+    id: 'filterstatus-id',
+
+    listeners: {
+
+        afterrender: function (panel) {
+
+            var hidden = !(this.filters && this.filters.length > 0) || !(this.selections && this.selections.length > 0);
+
+            if (hidden) {
+
+                this.hideGroupLabel();
+                this.hideGroupSavePanel();
+                this.resetFilterStatusBackGroundColor();
+                this.hideEditGroupBtn();
+                this.hideSaveAsGroupBtn();
+            }
+        }
+    },
+
+    hideGroupLabel() {
+        Ext.getCmp('savedgroupname-id').hide();
```

Comment:
Alternatively, if you are searching within the hierarchy of this container you can use the `itemId` with `Container.getComponent`.

Reply:
Ah I see, good to know! thanks!